vllm/entrypoints/openai/serving_transcription.py

```diff
@@ -195,7 +195,8 @@ async def _preprocess_transcription(
             raise ValueError("Maximum file size exceeded.")
 
         with io.BytesIO(audio_data) as bytes_:
-            y, sr = librosa.load(bytes_)
+            y, sr = librosa.load(bytes_, sr=16000)  # fixing to 16 kHz sr=16000
```

Comment:
Ok nice I assume this will do re-sampling in one go. We can get rid of the constant here once https://github.com/vllm-project/vllm/pull/13301/files#diff-e5b7e14ee8e9d570076278fd160967b165ec83a56ba8536940160bdce22c7dbbR166 lands